include/sot/torque_control/torque-offset-estimator.hh

```diff
@@ -79,7 +79,6 @@ namespace dynamicgraph {
 
         DECLARE_SIGNAL_IN(base6d_encoders,          dynamicgraph::Vector);
         DECLARE_SIGNAL_IN(accelerometer,            dynamicgraph::Vector);
-        DECLARE_SIGNAL_IN(gyroscope,                dynamicgraph::Vector);
```

Comment:
I'm not using it anywhere. Do you want to use the gyroscope as a check during run, such that if the check is breached, the program terminates?